### PR TITLE
Auto reset Azure DB every 1 hour.

### PR DESCRIPTION
**Pull request by isaacabraham**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -5,14 +5,25 @@
             "name": "Debug",
             "type": "coreclr",
             "request": "launch",
-            "preLaunchTask": "Start",
+            "preLaunchTask": "Build Server",
```
Comment (forki) [inline .vscode/launch.json]: Cc @Krzysztof-Cieslak

Comment (isaacabraham) [reply .vscode/launch.json]: Start didn't work for me when called through Code's debug / launch. But happy for that to be rolled back if needed.

**Comment by forki** [comment]
Can you please set the last db reset time as one mutable field or into the db? So that we can show that in UI

**Comment by isaacabraham** [comment]
@forki what should it give for the file system version? None?

**Comment by forki** [comment]
App startup?

Am 08.10.2017 15:54 schrieb "Isaac Abraham" <notifications@github.com>:

**Comment by isaacabraham** [comment]
@forki done - can you test it out locally (you can change local time to e.g. every 10 seconds in `Program.fs`.

**Comment by isaacabraham** [comment]
BTW: When type providers are properly supported and we port across the Azure TP, this code will be much nicer :-)

**Comment by forki** [comment]
Will do tomorrow. Thanks

**Pull request merged**